interface-definitions/containers.xml.in

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="container" owner="${vyos_conf_scripts_dir}/containers.py">
+    <properties>
+      <help>Container applications</help>
+    </properties>
+    <children>
+      <tagNode name="name">
+        <properties>
+          <help>Container name</help>
+        </properties>
+        <children>
+          <leafNode name="allow-host-networks">
+            <properties>
+              <help>Allow host networks in container</help>
+              <valueless/>
+            </properties>
+          </leafNode>
+          <leafNode name="description">
+            <properties>
+              <help>Container description</help>
+            </properties>
+          </leafNode>
+          <leafNode name="image">
+            <properties>
+              <help>Image name in the hub-registry</help>
+            </properties>
+          </leafNode>
+          <tagNode name="network">
+            <properties>
+              <help>Attach user defined network to container</help>
+              <completionHelp>
+                <path>container network</path>
+              </completionHelp>
+            </properties>
+            <children>
+              <leafNode name="address">
+                <properties>
+                  <help>Set IPv4 static address to container (optional)</help>
+                  <valueHelp>
+                    <format>ipv4</format> 
+                    <description>IPv4 address (x.x.x.1 reserved)</description>
+                  </valueHelp>
+                  <constraint>
+                    <validator name="ipv4-address"/>
+                  </constraint>
+                </properties>
+              </leafNode>
+            </children>
+          </tagNode>
+        </children>
+      </tagNode>
+      <tagNode name="network">
+        <properties>
+          <help>Network name</help>
+        </properties>
+        <children>
+          <leafNode name="description">
+            <properties>
+              <help>Network description</help>
+            </properties>
+          </leafNode>
+          <leafNode name="ipv4-prefix">
+            <properties>
+              <help>Prefix which allocated to that network</help>
+              <valueHelp>
+                <format>ipv4net</format>
+                <description>IPv4 network and prefix length</description>
+              </valueHelp>
+              <constraint>
+                <validator name="ip-prefix"/>
+              </constraint>
+            </properties>
+          </leafNode>
+          <leafNode name="type">
```

Comment:
If bridge is the only type possible, why do we need to specify this?

Reply:
> If bridge is the only type possible, why do we need to specify this?
Driver to manage the network (default "bridge"). Currently only bridge is supported.
https://github.com/containers/podman/blob/master/docs/source/markdown/podman-network-create.1.md

Reply:
Then please remove this option unöess something in addition is supported by podman.

Reply:
> Then please remove this option unöess something in addition is supported by podman.

Done